src/objects.js

```diff
@@ -1,42 +1,53 @@
 const createPerson = (name, age) => {
-  // your code here
+  return {name: name, age: age};
 };
 
 const getName = object => {
-  // your code here
+  return object.name;
 };
 
 const getProperty = (property, object) => {
-  // your code here
+  return object[property];
```

Comment:
This is a good example of a case where using square notation is needed over dot notation. You don't know which property you will be referencing from the object, so you want to be able to achieve this dynamically. Keep that in mind!